geode-docs/developing/function_exec/function_execution.html.md.erb

```diff
@@ -33,28 +33,30 @@ To write the function code, you implement the `Function` interface or extend the
 
```

Comment:
This bit of the docs looks to be out of date with the code.  `FunctionAdapter` is deprecated, so I think we ought to just say implement the `Function` without mentioning `FunctionAdapter`.  Also all of the default methods that used to live in `FunctionAdapter` are now Java 8 default methods on the `Function` interface.